pystan/model.py

```
@@ -54,20 +53,28 @@ def load_module(module_name, module_path):
         return imp.load_module(module_name, *module_info)
 
 
-def _map_parallel(function, args, n_jobs):
+def _map_parallel(function, args, n_jobs, n_threads=None):
     """multiprocessing.Pool(processors=n_jobs).map with some error checking"""
     # Following the error checking found in joblib
     multiprocessing = int(os.environ.get('JOBLIB_MULTIPROCESSING', 1)) or None
     if multiprocessing:
         try:
             import multiprocessing
             import multiprocessing.pool
+            if n_jobs < 1:
+                n_jobs = multiprocessing.cpu_count()
         except ImportError:
             multiprocessing = None
-        if sys.platform.startswith("win") and PY2:
-            msg = "Multiprocessing is not supported on Windows with Python 2.X. Setting n_jobs=1"
+            n_jobs = 1
```

We should raise an exception or somehow abort if we cannot use the user's arguments. Overriding a user's arguments is unusual.

---

yeah, this was my debug line, I removed it